src/login/Login.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Redirect } from "react-router-dom";
+import "../myspace/MySpace";
+
+
+export default function Login() {
+    const [username, setUsername] = useState(true);
+    const [password, setPassword] = useState(true);
+    const [responseMessage, setResponseMessage] = useState(null);
+    const [accessToken, setAccessToken] = useState(null);
+    const [accessExpiry, setAccessExpiry] = useState(null);
+    const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+    const handleSubmit = async e => {
+        e.preventDefault();
+        let userInput = {}
+
+        new FormData(e.target).forEach((value, key) => {
+            if (key === "username")
+                userInput[key] = value;
```

Comment:
Do you really need an if else statement here? The action that you are performing is the same either ways. 
Why can't it simply be userInput[key] = value?

Reply:
Yep, fixed